CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ endif()
 add_library(opentrustregion SHARED src/opentrustregion.f90 src/c_interface.f90)
 
 # enable preprocessing
-set_source_files_properties(src/opentrustregion.f90 PROPERTIES Fortran_PREPROCESS ON)
+set_source_files_properties(src/opentrustregion.f90 src/c_interface.f90 PROPERTIES Fortran_PREPROCESS ON)
```

Comment:
I think the suffix for preprocessed-to-be Fortran files is `.F90` instead of `.f90`.

However, since some people still use type-insensitive file systems, this might be the safer route.